### PR TITLE
[IMP] web: improve kanban tooltip

### PR DESCRIPTION
**Pull request by msh-odoo**
1) Remove the total record tooltip from kanban group.
    2) Remove progressbar help from kanban view.
    3) Improvements in testcase since we have removed total record tooltip.

Task:https://www.odoo.com/web#id=1845459&action=333&active_id=131&model=project.task&view_type=form&menu_id=4720
Pad: https://pad.odoo.com/p/r.7e13ec256c6c38a54652df07556bd309

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by msh-odoo** [comment]
LGTM

File: addons/web/static/src/js/views/kanban/kanban_column.js [hunk 1]
```diff
@@ -82,10 +82,12 @@ var KanbanColumn = Widget.extend({
 
         if (options.group_by_tooltip) {
             this.tooltipInfo = _.map(options.group_by_tooltip, function (help, field) {
-                return (data.tooltipData && data.tooltipData[field] && "<div>" + help + "<br>" + data.tooltipData[field] + "</div>") || '';
+                if (data.tooltipData && data.tooltipData[field]) {
+                    help = help ? help + "</br>" : help;
+                    data.tooltipData[field] = (!help && !_.values(data.tooltipData).includes(false)) ? data.tooltipData[field] + "<div class='dropdown-divider' role='separator' />" : data.tooltipData[field];
```
Comment (aab-odoo) [inline addons/web/static/src/js/views/kanban/kanban_column.js]: includes is ES6 (not supported by ie11), so you can't use it. You can use indexOf instead.
Also, this line is way too long and hard to read/understand, i would try to re-write it to make it more readable.

Comment (msh-odoo) [reply addons/web/static/src/js/views/kanban/kanban_column.js]: @aab-odoo It seems that something went wrong when @tivisse did rebase, I removed ```includes``` when reviewed it, I rebased branch again, you can check it now, I will make separate commit for web part, Thanks

**Comment by tivisse** [comment]
@robodoo r+

**Comment by tivisse** [comment]
@robodoo rebase-ff

**Pull request closed without merging**